src/main/java/gift/entity/Order.java

```diff
@@ -0,0 +1,60 @@
+package gift.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Long optionId;
```

Comment:
```suggestion
    @ManyToOne
    private Option option;
```
을 사용하시지 않은 이유가 궁금합니다~